### PR TITLE
unicode to ascii

### PR DESCRIPTION
**Pull request by baraahmad**

### CODE REVIEW

File: app/web-components/article.js [hunk 1]
```diff
@@ -1,12 +1,21 @@
 import React from 'react'
 import ArticleBlock from '../components/article-block'
 import { BrevoHelmet } from '../components/brevo-join'
+import { Helmet } from 'react-helmet'
+// import { ConvertUnicode } from '../components/convert-unicode'
```
Comment (ddfridley) [inline app/web-components/article.js:5]: please remove this comment since it's not relevant going forward.

**Review by ddfridley — CHANGES_REQUESTED** [review]
It works great. I appreciate that the unicode character got mapped into something.
Please remove the unrelated comment.
Also, there was no readme with contributing info so you wouldn't know this but please add a comment to the top of the file, linking to github issue you are working on.
And here's the contributing info: 
[Contribute](https://github.com/EnCiv/.github/wiki/Contributing)
[React Coding and Style Guidlines](https://github.com/EnCiv/.github/wiki/React-Coding-and-Style-Guidelines)

Thanks!

**Comment by baraahmad** [comment]
I removed the unnecessary comment you mentioned and included a link to the
github issue as a comment at the top of the article component.

**Review by ddfridley — APPROVED** [review]
Looks Great!

**Pull request merged**